tests/Fixer/ArrayNotation/AlphabeticalArrayKeySortFixerTest.php

```diff
@@ -0,0 +1,262 @@
+            ],
+            [
+                '<?php ["a" => array("l" => "10", "m" => "20"), "b" => "2", "d" => "5"];',
+                '<?php ["b" => "2", "a" => array("m" => "20", "l" => "10"), "d" => "5"];',
+            ],
+            [
+                '<?php ["a" => array(
+                        "l" => "10",
+                        "m" => "20"),
+                        "b" => "2",
+                        "d" => "5"
+                    ];',
+                '<?php ["b" => "2",
+                        "a" => array(
+                        "m" => "20",
+                        "l" => "10"),
+                        "d" => "5"
+                    ];',
+            ],
+            [
+                '<?php array("a" => array("l" => "10", "m" => "20", "z" => array("a" => "hame", "b" => "kame")), "b" => "2", "d" => "5");',
+                '<?php array("b" => "2", "a" => array("m" => "20", "z" => array("b" => "kame", "a" => "hame"), "l" => "10"), "d" => "5");',
+            ],
+            [
+                '<?php [1, "test1" => 0,2,3, "test2" => 0];',
+                '<?php [1, "test2" => 0,2,3, "test1" => 0];',
+            ],
+            [
+                '<?php [1,2,3, "test1" => 0, "test2" => 0];',
+                '<?php [1,2,3, "test2" => 0, "test1" => 0];',
+            ],
+            [
+                '<?php [["test1" => 0, "test2" => 0],["test1" => 0, "test2" => 0]];',
+                '<?php [["test2" => 0, "test1" => 0],["test2" => 0, "test1" => 0]];',
+            ],
+            [
+                '<?php [
+                        "a" => [
+                    "b" => [
+                        "c" => sprintf("%s %s", Arr::get($data, "name"), Arr::get($data, "last_name")),
+                        "d" => self::VARIABLE,
+                    ],
+                    "product" => [
+                        "Colour" => self::VARIABLE,
+                    ],
+                ],
+            ];',
+                '<?php [
+                        "a" => [
+                    "b" => [
+                        "d" => self::VARIABLE,
+                        "c" => sprintf("%s %s", Arr::get($data, "name"), Arr::get($data, "last_name")),
+                    ],
+                    "product" => [
+                        "Colour" => self::VARIABLE,
+                    ],
+                ],
+            ];',
+            ],
+            [
+                '<?php [
+                    "a" => $date->copy()->addDays(60)->setTime(12, 0, 0)->toDateTimeString(),
+
+                    // insert comment here.
+                    "b" => self::VARIABLE,
+                ];',
+                '<?php [
+                    "b" => self::VARIABLE,
+
+                    // insert comment here.
+                    "a" => $date->copy()->addDays(60)->setTime(12, 0, 0)->toDateTimeString(),
+                ];',
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider provideFixPhp70Cases
+     *
+     * @requires PHP 7.0
```

Comment:
@kubawerlos at the time, when we created the PR, it might've been different - to be honest that is the only reason I can come up with since it's been a while 😄 Was this the only remaining comment or should I wait before processing this?